apps/api/package.json

```diff
@@ -64,6 +64,9 @@
     "cron:portfolio-snapshot": "tsx scripts/take-portfolio-snapshots.ts",
     "cron:process-perps": "tsx scripts/process-perps-competitions.ts",
     "run:indexer": "tsx scripts/indexer.ts",
+    "indexer:run": "tsx scripts/indexer.ts",
```

Comment:
i guess we don't need this one. it's the same as `run:indexer`

Reply:
Ah, yeah, I can remove that.